mtgsqlive/__main__.py

```diff
@@ -26,14 +26,27 @@
         metavar="fileOut",
     )
     parser.add_argument(
-        "--all",
-        help="Build all types (SQLite, SQL, CSV)",
+        "-a",
```

Comment:
This is no longer backwards compatible. You can do "--all", "-a" to allow for both

Reply:
Should be all good now. I've made sure MySQL stays default and added long-hand and short-hand entries for all the flags so either will work, for consistency.